pkgs/top-level/aliases.nix

```diff
@@ -81,6 +81,7 @@ mapAliases ({
   amuleGui = throw "amuleGui was renamed to amule-gui"; # Added 2022-02-11
   amsn = throw "amsn has been removed due to being unmaintained"; # Added 2020-12-09
   angelfish = libsForQt5.kdeGear.angelfish; # Added 2021-10-06
+  angryipscanner = ipscan; # Added 2023-08-30
```

Comment:
Aliases do not work like this. They are only meant to be created for backwards compatibility and are going to be dropped in the future, hence the date.

Reply:
Thanks for the info, @SuperSandro2000!
How would I then best go about adding a proper alias for the package, as I imaging most people would search for `angryipscanner` when looking for the `ipscan` package?

Reply:
The most common package name should be `ipscan` [according to repology](https://repology.org/project/ipscan/versions), however?

Reply:
Yeah, I'm not saying we need to change the package's name, just that an alias would probably be helpful.

Reply:
I guess the alias should be moved to `pkgs/top-level/all-packages.nix` then. `pkgs/top-level/aliases.nix` is indeed for deprecation only.

Reply:
Thanks for the input! I fixed it in #252555